src/accelerate/utils/dataclasses.py

```diff
@@ -51,6 +51,28 @@ def to_kwargs(self):
         return {k: v for k, v in this_dict.items() if default_dict[k] != v}
 
 
+@dataclass
+class AutocastKwargs(KwargsHandler):
+    """
+    Use this object in your [`Accelerator`] to customize how `torch.autocast` behaves. Please refer to the
+    documentation of this [context manager](https://pytorch.org/docs/stable/amp.html#torch.autocast) for more
+    information on each argument.
+
+    Example:
+
+    ```python
+    from accelerate import Accelerator
+    from accelerate.utils import AutocastKwargs
+
+    kwargs = AutocastKwargs(cache_enabled=True)
+    accelerator = Accelerator(kwargs_handlers=[kwargs])
+    ```
+    """
+
+    enabled: bool = False
+    cache_enabled: bool = False
```

Comment:
Shouldn't the default value of ``cache_enabled`` be True, as that's the default value of pytorch

Reply:
The default value is `None`, but yes. I will change this. Nice catch.